funnel/extapi/explara.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+import requests
+from ..util import format_twitter
+from ..models import SyncTicket
+
+__all__ = ['ExplaraAPI']
+
+
+class ExplaraAPI(object):
+    def __init__(self, access_token):
+        self.access_token = access_token
+        self.headers = {'Authorization': u'Bearer ' + self.access_token}
+        self.base_url = 'https://www.explara.com/api/e/{0}'
+
+    def get_orders(self, explara_eventid):
+        ticket_orders = []
+        completed = False
+        from_record = 0
+        to_record = 50
+        while not completed:
+            payload = {'eventId': explara_eventid, 'fromRecord': from_record, 'toRecord': to_record}
+            attendee_response = requests.post(self.base_url.format('attendee-list'), headers=self.headers, data=payload).json()
```

Comment:
Don't use `self.base_url.format('attendee-list')`. Make this a separate function and use `self.url_for('attendee-list')`. That function can simply be:

```python
def url_for(self, endpoint):
    return 'https://www.explara.com/api/e/' + endpoint
```